pkg/ebpf/telemetry/errors_telemetry_test.go

```diff
@@ -0,0 +1,193 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package telemetry
+
+import (
+	"math"
+	"os"
+	"testing"
+
+	"golang.org/x/sys/unix"
+
+	"github.com/DataDog/datadog-agent/pkg/ebpf/bytecode"
+	manager "github.com/DataDog/ebpf-manager"
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+
+	"github.com/cilium/ebpf"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type bpfPrograms struct {
+	KprobeVfsOpen *ebpf.Program `ebpf:"kprobe__vfs_open"`
+}
+
+var m = &manager.Manager{
+	Probes: []*manager.Probe{
+		{
+			ProbeIdentificationPair: manager.ProbeIdentificationPair{
+				EBPFFuncName: "kprobe__vfs_open",
+			},
+		},
+	},
+	Maps: []*manager.Map{
+		{
+			Name: "error_map",
+		},
+		{
+			Name: "suppress_map",
+		},
+	},
+}
+
+func triggerTestAndGetTelemetry(t *testing.T) []prometheus.Metric {
+	bpfDir := os.Getenv("DD_SYSTEM_PROBE_BPF_DIR")
+	require.True(t, bpfDir != "")
```

Comment:
Why do you need this check?

Reply:
Incase `bpfDir` is not set. This could be a problem when running tests directly with `go test` for example.

Reply:
This is because you are directly reading the env var, rather than using the config which has a default value.

Reply:
Fixed